dkim-auth/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 
 
 [package]
-name = "contract"
-version = "1.0.0"
+name = "near-dkim-auth"
```

Comment:
Hmm, any reason you decided to include dkim in the name?

Reply:
I have two minor concerns about the name:
* We already have `dkim` package which has actual DKIM logic. This might be a little confusing.
* We have been using "DKIM" as our internal project name, presumably because it is nice and short. If we are going this route, I would think think that everything here should be prefixed with "dkim-": "dkim-delegation", "dkim-relayer" etc.

The reason why I like `controller` is the direct link/contrast between `controller` and `delegator`/`delegation`. But yeah we could make it less generic by naming it `auth-controller`/`dkim-controller` etc.

Reply:
Discussed on the call:
- delete near prefix from contracts
- rename `near-dkim-auth` to `dkim-controller`
- rename `access-delegation` - > `control-delegator`
- rename `owner_id` -> `controller_id`